app/assets/stylesheets/fonts.scss

```diff
@@ -0,0 +1,7 @@
+@import 'variables';
+
+@each $key, $value in $font-sizes {
+  .font-#{$key} {
+    font-size: $value;
+  }
```

Comment:
We have font size utilities available from Co-Design - are they not working for something here? If that's the case we should log an issue for it there and re-consider how they work.

Reply:
Yes, unfortunately, they are not - as you can see in the [typography section](https://design.codidact.org/utilities/typography/), these utilities are in absolute units _and_ do not match our usage in QPixel (the latter is the primary reason for the additions, though), so I ended up gathering `font-size` values scattered across QPixel's rulesets in one map.

These utilities should be moved to Co-Design, however, I feel like doing so now will end up stalling the PR for quite a while, so I'd keep them here until we can spare some bandwidth to move them over

Reply:
Absolutely, not suggesting we add more work here. Let's file it for later.